src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.GovNotifyEmailService
+import uk.gov.service.notify.NotificationClient
+
+/**
+ * Email configuration for the service. The sending of all emails is handled by the Gov Notify service.
+ *
+ * If the API key is present then emails will be sent via the Gov Notify service. If the API key is blank/not present
+ * then emails will not be sent.
+ */
+@Configuration
+class EmailConfiguration(
+  @Value("\${notify.api.key:}") private val apiKey: String,
+  @Value("\${notify.templates.prison.transferred:}") private val offenderTransferredPrisonEmailTemplateId: String,
+  @Value("\${notify.templates.court.transferred:}") private val offenderTransferredCourtEmailTemplateId: String,
+  @Value("\${notify.templates.prison.transferred-but-no-court-email:}") private val offenderTransferredPrisonEmailButNoCourtEmailTemplateId: String,
+  @Value("\${notify.templates.prison.released:}") private val offenderReleasedPrisonEmailTemplateId: String,
+  @Value("\${notify.templates.court.released:}") private val offenderReleasedCourtEmailTemplateId: String,
+  @Value("\${notify.templates.prison.released-but-no-court-email:}") private val offenderReleasedPrisonEmailButNoCourtEmailTemplateId: String,
+  @Value("\${notify.templates.appointment-canceled.prison:}") private val appointmentCanceledPrisonEmailTemplateId: String,
+  @Value("\${notify.templates.appointment-canceled.court:}") private val appointmentCanceledCourtEmailTemplateId: String,
+  @Value("\${notify.templates.appointment-canceled.no-court-email:}") private val appointmentCanceledPrisonEmailButNoCourtEmailTemplateId: String,
+) {
+
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @Bean
+  fun emailService() =
+    if (apiKey.isBlank()) {
```

Comment:
Trying to avoid loads of env vars e.g. is it enabled type thing.  The lack of key basically says sending of emails disabled.